doc/src/base/simd-types.md

```diff
@@ -10,7 +10,7 @@ end
 ```
 
 It has a special compilation rule: a homogeneous tuple of `VecElement{T}` maps to an LLVM `vector`
-type when `T` is a primitive bits type and the tuple length is in the set {2-6,8-10,16}.
```

Comment:
`T` still has to be a primitive bits type thought, right?

Reply:
I would imagine T still has to be a primitive bits type.